packages/dom/src/document.rs

```diff
@@ -739,31 +744,6 @@ impl Document {
         Some(cursor)
     }
 
-    pub fn scroll_by(&mut self, px: f64) {
-        // Invert scrolling on macos
-        #[cfg(target_os = "macos")]
-        {
-            self.scroll_offset += px;
-        }
-        #[cfg(not(target_os = "macos"))]
-        {
-            self.scroll_offset -= px;
-        }
```

Comment:
I think we still need this. This branch is scrolling in the incorrect direction for me. I am unsure if the inversion is macOS specific or scrolling needs to be inverted on all platforms.

Reply:
Ok, I realised it was scrolling the wrong way on my gnome desktop too, so in the updated scrolling code I've inverted it. Could you double check it's scrolling the right way on mac now?